### PR TITLE
Refactored Profile Views with hardcoded data

### PR DESCRIPTION
**Pull request by tiffanyyv**
Screen Shot 2022-03-23 at 1 36 18 PM

- Added in requests to fetch user info in NavLayout/Sidebar.js, ProfileView/ProfileMenu.js, [studentprofile], [teacherprofile], my-profile.js.  Will need to clean up these useEffects later.
- userID for these pages are currently hardcoded in.

To do:
- Need to figure out how to grab student's info when a teacher clicks on the student name to render the appropriate details in student profile view.
- Remove hardcoded teacher/student IDs.
- Add more styling to profile views

### CODE REVIEW

File: components/NavLayout/SideBar.js [hunk 1]
```diff
@@ -32,6 +43,15 @@ export default function SideBar({ children, ...props }) {
     setOpen(false);
   };
 
+  const fetchUserInfo = () => {
+    getUserInfo('51').then(res => {
+      console.log(res)
+      setCurrentUserProfileInfo({
+        fullName: res.name.first_name + ' ' + res.name.last_name,
```
Comment (cmjosephs) [inline components/NavLayout/SideBar.js:50]: Will need to add error handling for possible changes in API data. Handle later when we do the switch to dynamic routing, that way we can grab the uid from all pages. I'll also post an example of what error handling looks like in the components

File: components/features/ProfileView/ProfileMenu.js [hunk 2]
```diff
@@ -25,14 +35,23 @@ const ProfileMenu = () => {
     setAnchorUser(null);
   };
 
+  const fetchUserInfo = () => {
+    getUserInfo('51').then(res => {
+      setCurrentUserProfileInfo({
```
Comment (cmjosephs) [inline components/features/ProfileView/ProfileMenu.js:40]: Same as above

File: components/NavLayout/SideBar.js (hunk 1 above)
Comment (tiffanyyv) [reply components/NavLayout/SideBar.js:50]: Will do!

File: pages/app/my-profile.js [hunk 3]
```diff
@@ -1,30 +1,64 @@
...
-              className="my-profile-view-avatar"
-              alt="My Profile Picture"
-              src={defaultProfilePic}
-              sx={defaultProfilePicDims}
-            /></div>
+        <div className={styles.profilePic}>
+          <Avatar
+            className="my-profile-view-avatar"
+            alt="My Profile Picture"
+            src={currentUserProfileInfo.photo ? currentUserProfileInfo.photo : defaultProfilePic}
```
Comment (cmjosephs) [inline pages/app/my-profile.js:51]: Good job handling for no provided profile photo

**Review by cmjosephs — APPROVED** [review]
Looks good overall. A few things to refactor but that will come later I think.

**Comment by cmjosephs** [comment]
@tiffanyyv I approved it. Go ahead and merge when you're ready unless you want more code review

**Pull request merged**